cmd/kops/create_cluster.go

```diff
@@ -800,9 +800,23 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		}
 	}
 
+	// Populate project
 	if c.Project != "" {
 		cluster.Spec.Project = c.Project
 	}
+	if api.CloudProviderID(cluster.Spec.CloudProvider) == api.CloudProviderGCE {
+		if cluster.Spec.Project == "" {
+			project, err := gce.DefaultProject()
+			if err != nil {
+				glog.Warningf("unable to get default google cloud project: %v", err)
+			} else if project == "" {
+				glog.Warningf("default google cloud project not set")
```

Comment:
From a ux perspective should we tell them which flag to use? In your words the error could be more actionable. Also this is an error not a warning.